src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMockFactory.java

```diff
@@ -1,67 +1,45 @@
 package be.seeseemelk.mockbukkit.entity;
 
-import java.util.HashSet;
-import java.util.Random;
-import java.util.Set;
-import java.util.UUID;
-
 import be.seeseemelk.mockbukkit.ServerMock;
+import org.jetbrains.annotations.NotNull;
 
-public final class PlayerMockFactory
-{
-	private static final String[] FIRST_NAMES = { "James", "Mary", "John", "Particia", "Robert", "Jennifer", "Michael", "Elizabeth", "William", "Linda" };
-	private static final String[] LAST_NAMES = { "Smith", "Johnson", "Jones", "Brown", "Davis", "Miller", "Wilson", "Moore", "Taylor", "Anderson" };
+import java.util.Random;
+import java.util.UUID;
 
+public final class PlayerMockFactory {
 	private final ServerMock server;
-	private Random random = new Random();
-	private Set<String> usedNames = new HashSet<>();
+	private final Random random = new Random();
+	private int currentNameIndex;
```

Comment:
```suggestion
	private long currentNameIndex;
```
int type would limit us to use only 2147483647 players, because after this number currentNameIndex would overflow

Reply:
Indeed, this would make it so that the exception at line 31 could never be thrown, because Player2147483647 is exactly 16 characters long. That said, I don't think it is realistic anyone would ever create that many players.